packages/back-end/src/api/openapi/openapi.yaml

```
@@ -115,7 +115,12 @@ paths:
   /data-sources/{id}:
     $ref: "./paths/getDataSource.yaml"
   /experiments:
-    $ref: "./paths/listExperiments.yaml"
+    get:
+      $ref: "./paths/listExperiments.yaml"
+    post:
+      $ref: "./paths/postExperiment.yaml"
+    put:
+      $ref: "./paths/putExperiment.yaml"
   /experiments/{id}:
     $ref: "./paths/getExperiment.yaml"
```

This needs to change to something like:

```suggestion
    get:
      $ref: "./paths/listExperiments.yaml"
    post:
      $ref: "./paths/postExperiment.yaml"
  /experiments/{id}:
    get:
      $ref: "./paths/getExperiment.yaml"
    post:
      $ref: "./paths/putExperiment.yaml"
```